package.json

```diff
@@ -1,9 +1,11 @@
 {
-  "name": "alarms-trigger",
-  "version": "1.0.0",
-  "description": "A service to trigger openwhisk triggers based on cron notation",
+  "name": "@openwhisk/alarms-trigger",
```

Comment:
As far as I can tell, we've never actually published this package on npm.  So let's change the name to go to the openwhisk organization so that if we do decide to publish it at some point, it will go to a reasonable place.